### DISCUSSION TITLE
Maps

### DISCUSSION THREAD

**Discussion by Sheduw** [TEHE-Studios/ExpandedHelicopterEvents#183] [Q&A]
Hi! I'm writing this to ask, how did you guys make the custom maps that you can read? I always wanted to make one myself, but I wasn't sure how exactly!

**Comment by Chuckleberry-Finn** [comment]
This is a question regarding mod making and not really one about the game-play of this mod - that being said you can access the mod's files from your local content from the workshop. If this is reference to the fliers, which are modified 'map' items.
